src/components/elementView/elementTab.tsx

```diff
@@ -488,6 +487,15 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
               onClick={this.handleOpen}
             />
           ) : null}
+          {this.props.element === ElementType.ASSET ? (
+            <AnchorButton
+              className="add"
+              text="Print Barcodes for Selected Assets"
+              icon="barcode"
+              minimal
+              onClick={() => {}}
```

Comment:
I presume this onClick part here is where I come in with back end stuff to get the barcodes for all assets currently filtered?